SwiftyBluetooth/Source/Central.swift

```diff
@@ -144,6 +156,20 @@ extension Central {
         return self.centralProxy.centralManager.isScanning
     }
     
+    /// Alternative connection method, when identifier and services are known for a peripheral.
+    ///
+    /// - Parameter peripheralUUID: The unique identifier (UUID) for the given peripheral.
+    /// - Parameter serviceUUIDs: The service UUIDs to for the given peripheral.
+    /// - Parameter timeout: The time in seconds before the connection attempt is stopped and the completion closure is called with a failed result.
+    /// - Parameter completion: The closures, called upon completion (succesful or otherwise) of the connection.
+    public func connect(peripheralUUID: UUID,
```

Comment:
@tehjord I'm still a bit confused, if this library is meant to just be a light wrapper for the CoreBluetooth API, why has the connect method been made something that is only accessible from the Peripheral class, not from the Central class as it is with Apple's API.